src/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable max-len */
+import classNames from 'classnames';
+
+type Props = {
+  setError: (error: boolean) => void;
+  error: boolean;
+};
+
+export const ErrorNotification: React.FC<Props> = ({ setError, error }) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames('notification is-danger is-light has-text-weight-normal', {
+        'notification is-danger is-light has-text-weight-normal hidden': !error,
+      })}
```

Comment:
```suggestion
      className={classNames(
      'notification is-danger is-light has-text-weight-normal', 
        {
          hidden: !error,
      })}
```

Reply:
classnames - always add classes which passed as a string, and classes from object can be added depend on condition/